controller/openid_configuration_blackbox_test.go

```diff
@@ -0,0 +1,77 @@
+package controller_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-auth/app"
+	"github.com/fabric8-services/fabric8-auth/app/test"
+	"github.com/fabric8-services/fabric8-auth/client"
+	. "github.com/fabric8-services/fabric8-auth/controller"
+	"github.com/fabric8-services/fabric8-auth/resource"
+	"github.com/fabric8-services/fabric8-auth/rest"
+
+	"github.com/goadesign/goa"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type TestOpenIDConfigurationREST struct {
+	suite.Suite
+}
+
+func TestRunOpenIDConfigurationREST(t *testing.T) {
+	resource.Require(t, resource.UnitTest)
+	suite.Run(t, &TestOpenIDConfigurationREST{})
+}
+
+func (s *TestOpenIDConfigurationREST) UnSecuredController() (*goa.Service, *OpenidConfigurationController) {
+	svc := goa.New("Status-Service")
+	return svc, NewOpenidConfigurationController(svc)
+}
+
+func (s *TestOpenIDConfigurationREST) TestShowOpenIDConfiguration() {
+	t := s.T()
+	svc, ctrl := s.UnSecuredController()
+
+	_, openIDConfiguration := test.ShowOpenidConfigurationOK(t, svc.Context, svc, ctrl)
+
+	u := &url.URL{
```

Comment:
You don't need all these lines (43-61), do you?
Why not just compare expected hardcoded values like authorization endpoint "http:///api/authorize" with the returned configuration, etc.

Reply:
issuer would be a variable here (auth.openshift.io or localhost:8089 etc). If I were to remove that I would only be able to check for example if `/api/authorize` is included in authorizationEndpoint

Reply:
Host is always an empty string in the tests, right?

Reply:
Ok, Now I know that        :)